src/evo/netinfo.h

```diff
@@ -60,6 +60,35 @@ constexpr std::string_view NISToString(const NetInfoStatus code)
     assert(false);
 }
 
+// Purpose corresponds to the index position in the ExtNetInfo map (which is ordered),
+// entries must be contiguous and cannot be changed once set without a format version
+// update, which will necessitate a hard-fork.
+namespace Purpose {
+enum : uint8_t {
```

Comment:
nit: `enum class` for all new code is preferable.

Reply:
Resolved in latest push.